lib/matplotlib/tests/test_triangulation.py

```diff
@@ -1177,3 +1177,10 @@ def test_tricontourset_reuse():
     assert tcs2._contour_generator != tcs1._contour_generator
     tcs3 = ax.tricontour(tcs1, z)
     assert tcs3._contour_generator == tcs1._contour_generator
+
+
+@check_figures_equal()
+def test_triplot_with_ls(fig_test, fig_ref):
+    fig_test.subplots().triplot([0, 2, 1], [0, 0, 1], [[0, 1, 2]], ls='--')
+    fig_ref.subplots().triplot([0, 2, 1], [0, 0, 1], [
+        [0, 1, 2]], linestyle='--')
```

Comment:
I would re-factor the data out so it's clear that it's the same (and also no wrapping would be needed.)

Reply:
Thanks .
I didn't actually get what you meant, do I need to do any changes or anything like that?

Reply:
Something like
```suggestion
    x = [0, 2, 1]
    y = [0, 0, 1]
    data = [[0, 1, 2]]
    fig_test.subplots().triplot(x, y, data, ls='--')
    fig_ref.subplots().triplot(x, y, data, linestyle='--')
```

Reply:
Thanks. 👍 Made the changes .I will take care of it from now onwards.